nuxeo-admin-console-web/angular-app/src/app/shared/services/shared-methods.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { CustomSnackBarComponent } from "../components/custom-snack-bar/custom-snack-bar.component";
+import { MatDialog, MatDialogRef } from "@angular/material/dialog";
+import { ErrorModalComponent } from "../../features/sub-features/generic-multi-feature-layout/components/error-modal/error-modal.component";
+import { MODAL_DIMENSIONS } from "../../features/sub-features/generic-multi-feature-layout/generic-multi-feature-layout.constants";
+import { ErrorModalClosedInfo, ErrorDetails } from "../types/common.interface";
+
+/** This service handles common methods such as snack bars and dialogs.
+ * It helps avoid code duplication. */
+
```

Comment:
Are we using these shared methods in all the old features like reindexing and the new ones too ?